lambda_backend/predict_microservice/handler.py

```diff
@@ -13,7 +13,7 @@
 
 # Load Yolo
 net = cv2.dnn.readNet(
-    join("weights", "yolov4-tiny-obj_2000.weights"),
+    join("weights", "yolov4-tiny-obj.weights"),
     join("weights", "yolov4-tiny-obj.cfg")
 )
 
```

Comment:
for classes
```
classes = ["hold"]
```
And write a comment about referring to the colab notebook for the index to class mappings